docs/source/guides/writer/chapters/basics.rst

```diff
@@ -11,3 +12,60 @@ Notice that the file is given executable permissions, which is a
 requirement for Avocado to treat it as a executable test. Also notice
 that the script exits with status code 0, which signals a successful
 result to Avocado.
+
+SIMPLE Tests BASH extensions
```

Comment:
This may be a good time to rename this section.  We tried to eliminate the ```SIMPLE``` nomenclature with the legacy runner.